pkg/handlers/handler_base.go

```diff
@@ -0,0 +1,171 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/facebookgo/clock"
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+)
+
+// NewHandlerBase is a constructor for HandlerBase
+func NewHandlerBase(logger *zap.Logger) HandlerBase {
+	return HandlerBase{
+		logger: logger,
+		clock:  clock.New(),
+	}
+}
+
+// HandlerBase is for shared handler utilities
+type HandlerBase struct {
+	logger *zap.Logger
+	clock  clock.Clock
+}
+
+type errorItem struct {
+	Field   string `json:"field"`
+	Message string `json:"message"`
+}
+
+// errorResponse contains the structure of error for a http response
+type errorResponse struct {
```

Comment:
Don't get too caught up on this structure. It's easy to change and I really just want to get something in there for now. If other folks have thoughts on a standardized error format, they're absolutely welcome. I just don't think they're required for this PR.